bincopy.py

```diff
@@ -198,6 +201,9 @@ def is_ihex(records):
     else:
         return True
 
+def is_ti_txt(records):
+    return records[0] in ('@', 'q')
+
```

Comment:
I tend to use lists instead of tuples when checking is an iterable contains an item. I suggest changing to `['@', 'q']`.